paper.tex

```diff
@@ -78,6 +78,10 @@ \subsection{Generative models}
 
 The exact procedure how DALL-E models work proposed by Ramesh et. al is explained in their paper. \cite{ramesh2021zeroshot}
 
+Another image generative model is the \textbf{midjourney} \cite{midjourney}. This model was created by an independent research lab with the same name. It can synthesize images from textual descriptions. Midjourney tends to generate surrealistic images and is popular among artists and it uses modern deep learning methods to create high-quality images.
```

Comment:
```suggestion
Another image generative model is \textbf{midjourney} \cite{midjourney}. This model was created by an independent research lab with the same name. It can synthesize images from textual descriptions. Midjourney tends to generate surrealistic images and is popular among artists. It uses modern deep learning methods to create high-quality images.
```